src/plugins/notebook/components/Notebook.vue

```diff
@@ -419,43 +434,29 @@ export default {
             this.sidebarCoversEntries = sidebarCoversEntries;
         },
         getDefaultPageId() {
-            let defaultPageId;
-
-            if (this.isDefaultNotebook()) {
-                defaultPageId = getDefaultNotebook().page.id;
-            } else {
-                const firstSection = this.getSections()[0];
-                defaultPageId = firstSection && firstSection.pages[0].id;
-            }
-
-            return defaultPageId;
+            return this.isDefaultNotebook()
+                ? getDefaultNotebook().defaultPageId
+                : undefined;
         },
         isDefaultNotebook() {
             const defaultNotebook = getDefaultNotebook();
-            const defaultNotebookIdentifier = defaultNotebook && defaultNotebook.notebookMeta.identifier;
+            const defaultNotebookIdentifier = defaultNotebook && defaultNotebook.identifier;
 
             return defaultNotebookIdentifier !== null
                 && this.openmct.objects.areIdsEqual(defaultNotebookIdentifier, this.domainObject.identifier);
         },
         getDefaultSectionId() {
-            let defaultSectionId;
-
-            if (this.isDefaultNotebook()) {
-                defaultSectionId = getDefaultNotebook().section.id;
-            } else {
-                const firstSection = this.getSections()[0];
-                defaultSectionId = firstSection && firstSection.id;
-            }
-
-            return defaultSectionId;
+            return this.isDefaultNotebook()
+                ? getDefaultNotebook().defaultSectionId
+                : undefined;
         },
         getDefaultNotebookObject() {
             const oldNotebookStorage = getDefaultNotebook();
```

Comment:
This is me nit picking but could we change the 'getDefaultNotebook()' utility name to indicate that's it's the one from local storage somehow? It's been tripping me up the whole time I reviewed this PR :)

Reply:
Yeah I think ll change that in other PR. 😄